packages/notification-center/src/components/notification-center-web-component/notification-center-component.tsx

```diff
@@ -107,7 +107,8 @@ function PopoverWrapper({
   unseenBadgeColor,
   unseenBadgeBackgroundColor,
 }: PopoverWrapperProps) {
-  const { updateAction } = useUpdateAction();
+  const { storeQuery } = useNotifications();
+  const { updateAction } = useUpdateAction({ query: storeQuery });
```

Comment:
Not sure is it is the best way to do so, but at the moment we need to pass the query to the cache key..